packages/@uppy/tus/src/index.js

```diff
@@ -252,6 +259,24 @@ module.exports = class Tus extends BasePlugin {
         resolve(upload)
       }
 
+      uploadOptions.onShouldRetry = (err, retryAttempt, options) => {
+        if (err?.originalResponse?.getStatus() === 429) {
+          if (!this.requests.isPaused) {
```

Comment:
Without knowing the full API ... an `isPaused` on a plural noun seems fishy. I'd expect `this.requests` to be a collection of request objects that would each have an `isPaused` property. I guess it's for ease-of-use/simplification to have the `isPaused` property on the collection and then `pause()` and `resume()` all requests at the same time.

Reply:
`this.requests` is a `RateLimitedQueue` object, it's unlucky that it hasn't been named `this.requestQueue` or something like that instead. Because it's publicly exposed, we cannot change it until the next semver-major.